tuned/plugins/base.py

```diff
@@ -455,9 +464,16 @@ def _verify_all_device_commands(self, instance, devices, ignore_missing):
 			new_value = instance.options.get(command["name"], None)
 			if new_value is None:
 				continue
+			dvs = []
 			for device in devices:
+				ret = True
```

Comment:
Yes, here I set ret=True, because while traversing devices say cpu0, cpu1, cpu2 ...
if _verify_device_command() returns False for 'cpu1', 'ret' stays False for subsequent
'cpu2', 'cpu3' even if _verify_device_command() returned True.

Reply:
This is the same as my comment to line 452.

Reply:
Moved vlog.add_log() and related changes to the _verify_value() function should address this too.